examples_esp/getting_started/.gitignore

```diff
@@ -13,4 +13,7 @@ dependencies.lock
 
 # generated ESP-IDF config files
 sdkconfig.*
-!sdkconfig.defaults
\ No newline at end of file
+!sdkconfig.defaults
+!sdkconfig.defaults.*
+
+platformio_override.ini
```

Comment:
I think this `platformio_override.ini` is specific to openDTU project, yes?. We're not using any inherited PIO config here

Reply:
See below.